l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -498,13 +498,13 @@ def setDatiGeneraliDocumento(self, invoice, body):
                 if not causale:
                     continue
                 causale_list_200 = \
-                    [causale[i:i+200] for i in range(0, len(causale), 200)]
+                    [causale[i:i + 200] for i in range(0, len(causale), 200)]
```

Comment:
Come mai anche queste modifiche?

Reply:
Nessun Motivo in particolare, ho fatto il reformat code e automaticamente mi ha inserito questi 2 spazi..